src/app/home/home-gallery/home-gallery.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-home-gallery',
+  templateUrl: './home-gallery.component.html',
+  styleUrls: ['./home-gallery.component.css']
+})
+export class HomeGalleryComponent implements OnInit {
+
+  pictureLink= '/assets/img/resImage_300x260.jpg' ;
+  pictureArray : string[]= [];
+
+  constructor() { }
+
+  ngOnInit() {
+
+    for(var i=0; i<9; i++)
```

Comment:
Do not hardcode the counter in the for-loop.

bad: `for(var i=0; i<9; i++)` 👎 

good `for(var i=0; i<pictureArray.length; i++)` 👍